src/styles/blocks/card.scss

```diff
@@ -0,0 +1,139 @@
+.card {
+  box-sizing: border-box;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  width: $main-card-width + 100px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  padding-top: 32px;
+
+  font-family: 'Roboto ', sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+
+  transition: $main-transition;
+
+  &__img {
+    width: $width-img;
+    height: $hight-img;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    width: $width-content;
+    font-size: 1.2em;
+    font-weight: 500;
+    line-height: 18px;
+    color: rgba(6, 11, 53, 1);
+
+    margin: 0 0 4px;
+
+    transition: $main-transition;
+  }
+
+  &__description {
+    width: $width-content;
+    font-weight: 400;
+    font-size: 1em;
+    line-height: 14px;
+    color: rgba(97, 96, 112, 1);
+
+    margin: 0;
+    margin-bottom: $main-margin-bt;
+  }
+
+  &__rating {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: $width-content;
+
+    margin-bottom: 24px;
+
+    &-text {
+      margin: 0;
+      font-size: 1em;
+      line-height: 14px;
+      color: rgba(6, 11, 53, 1);
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    width: $width-content;
+
+    margin-bottom: $main-margin-bt;
+
+    &-text {
+      margin: 0;
+      font-size: 1.2em;
+      line-height: 18px;
+      color: rgba(97, 96, 112, 1);
+    }
+
+    &-value {
+      margin: 0;
+      font-weight: 700;
+      font-size: 1.6em;
+      line-height: 18px;
+    }
+  }
+
+  &__button {
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    width: $width-content;
+    height: 40px;
+
+    border: 1px solid $main-btn-color;
+    border-radius: 5px;
+    background: $main-btn-color;
+    transition: $main-transition;
+
+    text-decoration: none;
+    text-transform: uppercase;
+    color: rgba(255, 255, 255, 1);
+    font-size: 1.4em;
+    font-weight: 700;
+    line-height: 16px;
```

Comment:
Ensure that the font-size is consistent with the project's code style rules. If 'em' units are not specified in the project's design, consider using 'px' or 'rem'.